### PR TITLE
Language update from LingoHub 🤖 on 2020-12-26Z

### PR DESCRIPTION
**Pull request by norbertschuler**
Project Name: Treemapper App (Client)
Project Link: https://translate.lingohub.com/plant-for-the-planet/dashboard/treemapper-app-(client)
User: Norbert Schuler

### CODE REVIEW

**Comment by norbertschuler** [comment]
@sagararyal Is there any reason, why we do not merge this resource update? I cannot do it as the PR needs approval by someone else.

**Review by sagararyal — APPROVED** [review]

**Pull request merged**
